src/common/symbols.map

```diff
@@ -513,4 +512,5 @@ MIR_COMMON_2.9 {
     vtable?for?mir::logging::FileLogger;
     vtable?for?mir::logging::MultiLogger;
   };
+  local: *;
```

Comment:
Why? We don't do that elsewhere

Reply:
https://mir-server.io/doc/dso_versioning_guide.html#autotoc_md15 says:

> Note that the script is read top to bottom; wildcards are greedily bound when first encountered, so to avoid surprises you should only have a wildcard in the final stanza.

Reply:
Hmm:

* regenerate-miral-symbols-map.py
* src/miral/symbols.map
* src/platform/symbols.map
* src/wayland/symbols.map

Reply:
:man_shrugging: undid, then.